TaskManager/Home/View/DataSourceDiffable.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DataSourceDiffable.swift
+//  TaskManager
+//
+//  Created by user on 15.09.2022.
+//
+
+import Foundation
+import UIKit
+
+class DataSourceDiffable: UITableViewDiffableDataSource<Section, ModelTask> {
+    
+    let sectionArray = ["Active", "Completed"]
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return sectionArray[section]
```

Comment:
snapshot().sectionIdentifiers[section]
we use diffable no need to use other data when we already configured it